release/tasks/update-configuration-all.ts

```diff
@@ -0,0 +1,100 @@
+import * as path from 'path';
+
+import * as globby from 'globby';
+import * as fs from 'fs-extra';
+
+import { Context, Bump, Package } from '../@types/custom';
+import { readFile } from '../lib/utils';
+import calculatePackageNewVersion from '../lib/calculate-package-new-version';
+
+const ignorePrefixes = ['@hint/configuration-', '@hint/connector-chrome', 'create-', '@hint/utils-'];
+
+const ignorePackage = (pkg: Package): boolean => {
+    if (pkg.ignore) {
+        return true;
+    }
+
+    return ignorePrefixes.some((prefix) => {
+        return pkg.name.startsWith(prefix);
+    });
+};
+
+const isHint = (pkg: Package): boolean => {
+    return pkg.name.startsWith('@hint/hint-');
+};
+
+const getHints = async (pkg: Package) => {
+    const hints: any = {};
+
+    const metasPromises = (await globby(['src/meta.ts', 'src/meta/*.ts'], {
+        absolute: true,
+        cwd: path.join(pkg.path, '..')
+    })).map((meta) => {
+        return readFile(meta);
+    });
+
+    const metas = await Promise.all(metasPromises);
+
+    for (const meta of metas) {
+        const hintIdRegex = /id:\s*'([^']*)'/;
+        const id = meta.match(hintIdRegex);
+
+        if (id) {
+            hints[id[1]] = 'error';
+        }
+    }
+
+    return hints;
+};
+
+/**
+ * Bumps the version of each package:
+ *
+ * 1. Load current `configuration-all` package.
+ * 2. Load current `index.json`.
+ * 3. Clean current dependencies in `package.json`.
+ * 4. Clean current hints in `index.json`.
+ * 5. Add packages to `package.json` and rules to `index.json`.
+ * 6. Save `index.json`.
+ */
+export const updateConfigurationAll = (ignoredPackages: string[]) => {
```

Comment:
Doesn't look like `ignoredPackages` is used anywhere in the implementation. Was it meant to be used somewhere?

Reply:
good catch!